charts/consul/templates/terminating-gateways-deployment.yaml

```diff
@@ -345,6 +350,10 @@ spec:
                       -partition={{ $root.Values.global.adminPartitions.name }} \
                       {{- end }}
                       -id="${POD_NAME}"
+                  {{- if $root.Values.global.acls.manageSystemACLs }}
+                  - |
+                    consul-k8s-control-plane consul-logout
```

Comment:
The `consul-k8s-control-plane binary` will not be available on this container as this is an envoy container. you will have to issues a `/consul-bin/consul logout`. Something similar is done in the mesh gateway. Also, I like the way it is done here as a new entry into the slice. If this templating does work, can we update the mesh gateway to do something similar? I have used some `&&` logic there but this looks cleaner.